extensions/wikia/CuratedContent/js/CuratedContentManagmentTool.js

```diff
@@ -380,16 +384,38 @@ require(['wikia.window', 'jquery', 'wikia.nirvana', 'wikia.tracker', 'JSMessages
 												default:
 													$itemWithError = $(this);
 											}
+											if ($itemWithError) {
+												$itemWithError.addError(reasonMessage);
+												return false;
+											}
+										}
+										return true;
+									});
 
-											$itemWithError.addError(reasonMessage);
-											return false;
+									$sections.each(function () {
+										if (this.value === errTitle) {
+											var $itemWithError;
+
+											switch (errReason) {
+												case 'missingImage':
+													$itemWithError = $(this).parent().find('.image');
+													break;
+												case 'duplicatedLabel':
+												case 'tooLongLabel':
+													$itemWithError = $(this);
+													break;
+											}
+											if ($itemWithError) {
+												$itemWithError.addError(reasonMessage);
+												return false;
```

Comment:
Does it mean that we break from `$sections.each` loop and display only the first error?